docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -151,13 +173,20 @@ public class AdminResource {
     }
 }
 ----
+====
++
+[NOTE]
+====
+The main difference in this example is that the `@RolesAllowed` annotation is used to verify that only users granted with the `admin` role can access the endpoint.
```

Comment:
```suggestion
The main difference in this example is that the `@RolesAllowed` annotation is used to verify that only users granted the `admin` role can access the endpoint.
```

Reply:
or the `_users with the `admin` role can access._` to be consistent with my first suggestion.

Reply:
```suggestion
The main difference in this example is that the `@RolesAllowed` annotation is used to verify that only users with the `admin` role can access the endpoint.
```